keg_bouncer/tokens.py

```diff
@@ -20,13 +25,14 @@ def __init__(self, secret, timestamp_signer=TimestampSigner):
             key = secret + precursor
         else:
             key = secret.encode("utf-8") + precursor
-        self.cipher = AES.new(key[0:16])
+        self.cipher = Cipher(cipher_algos.AES(key[:16]), cipher_modes.ECB(), crypto_backend())
```

Comment:
More importantly, why not use the "safer" and one step fernet?

See https://cryptography.io/en/latest/fernet/